client/my-sites/plans-features-main/index.tsx

```diff
@@ -161,6 +161,44 @@ const OnboardingPricingGrid2023 = ( props: OnboardingPricingGrid2023Props ) => {
 		};
 	}
 
+	const [ masterbarHeight, setMasterbarHeight ] = useState( 0 );
+
+	/**
+	 * Calculates the height of the masterbar if it exists, and passes it to the component as an offset
+	 * for the sticky CTA bar.
+	 */
+	useLayoutEffect( () => {
```

Comment:
Thanks for the helpful comment here Aneesh. I'm wondering what the reasoning was to go with DOM calculations instead of an offset through CSS?

It looks like there are discrete breakpoints where the masterbar changes which, in my mind, we could represent as media queries